src/encoded/static/components/cart/cart.js

```diff
@@ -1556,6 +1571,67 @@ const processFilesAnalyses = (files, analyses) => {
 };
 
 
+/**
+ * Mutate the files of the given datasets that have no files with preferred_default set so that
+ * they appear as though they had preferred_default set. Once wranglers have patched all
+ * appropriate files' preferred_default properties, this function serves no purpose. A number of
+ * criteria determine how and whether any files within a dataset get this modification. Exported
+ * for Jest test.
+ * @param {array} datasets Datasets containing files to mutate with preferred_default
+ */
+export const processPseudoDefaultFiles = (datasets = []) => {
```

Comment:
"Pseudo" and your comments imply to me that the function **processPseudoDefaultFiles**  
 will get a name and functionality change. If that is true, would it be a good idea to name it as if the functionality will not change? Plus, sometimes, temporary last as long as permanent.